app/client/src/components/shared/WaterbodyInfo.tsx

```diff
@@ -519,11 +609,95 @@ function WaterbodyInfo({
                     return (
                       <tr key={useField.value}>
                         <td>
+                          <Modal
+                            label={`Detailed Uses for ${useField.label}`}
+                            maxWidth="35rem"
+                            onClose={() => setSelectedUseField(null)}
+                            triggerElm={
+                              <i
+                                className="fas fa-question-circle"
+                                css={detailedUsesIconStyles}
+                                title={`View detailed uses for ${useField.label}`}
+                                onClick={() => {
+                                  setSelectedUseField(useField);
+                                  fetchDetailedUses();
+                                }}
+                              ></i>
+                            }
+                          >
+                            {useAttainments.status === 'fetching' && (
+                              <LoadingSpinner />
+                            )}
+
+                            {selectedUseField &&
+                              useAttainments.status === 'success' && (
+                                <table
+                                  css={measurementTableStyles}
```

Comment:
Personally, I think this content would look cleaner if the top border and bottom margin were knocked off. So this could be:
```suggestion
                                  css={modalTableStyles}
```
which might be defined as:
```
const modalTableStyles = css`
  ${measurementTableStyles};
  margin-bottom: 0;

  th {
    border-top: none;
  }
`;
```

Reply:
Good call! It definitely looks better without the top border.